### ISSUE TITLE
Docs do not build in parallel

### ISSUE THREAD

**Issue by QuLogic** [matplotlib/matplotlib#16810]
**Bug summary**

Instructions for making a release say to run with `O=-j$(nproc)`, but running in parallel causes random plots to fail.

**Code for reproduction**
```
$ make -Cdoc O=-j4 html
```

**Actual outcome**

```
lib/matplotlib/axes/_axes.py:docstring of matplotlib.axes.Axes.secondary_xaxis:52: WARNING: Exception occurred in plotting matplotlib-axes-Axes-secondary_xaxis-1
 from doc/api/_as_gen/matplotlib.axes.Axes.secondary_xaxis.rst:
Traceback (most recent call last):
  File "lib/matplotlib/sphinxext/plot_directive.py", line 612, in render_figures
    figman.canvas.figure.savefig(img.filename(fmt), dpi=dpi)
  File "lib/matplotlib/figure.py", line 2203, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "lib/matplotlib/backend_bases.py", line 2105, in print_figure
    **kwargs)
  File "lib/matplotlib/backends/backend_pdf.py", line 2548, in print_pdf
    self.figure.draw(renderer)
  File "lib/matplotlib/artist.py", line 38, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "lib/matplotlib/figure.py", line 1736, in draw
    renderer, self, artists, self.suppressComposite)
  File "lib/matplotlib/image.py", line 137, in _draw_list_compositing_images
    a.draw(renderer)
  File "lib/matplotlib/artist.py", line 38, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "lib/matplotlib/axes/_base.py", line 2630, in draw
    mimage._draw_list_compositing_images(renderer, self, artists)
  File "lib/matplotlib/image.py", line 137, in _draw_list_compositing_images
    a.draw(renderer)
  File "lib/matplotlib/artist.py", line 38, in draw_wrapper
    return draw(artist, renderer, *args, **kwargs)
  File "lib/matplotlib/axis.py", line 1229, in draw
    renderer)
  File "lib/matplotlib/axis.py", line 1174, in _get_tick_bboxes
    for tick in ticks if tick.label1.get_visible()],
  File "lib/matplotlib/axis.py", line 1174, in <listcomp>
    for tick in ticks if tick.label1.get_visible()],
  File "lib/matplotlib/text.py", line 905, in get_window_extent
    bbox, info, descent = self._get_layout(self._renderer)
  File "lib/matplotlib/text.py", line 293, in _get_layout
    ismath="TeX" if self.get_usetex() else False)
  File "lib/matplotlib/backends/_backend_pdf_ps.py", line 62, in get_text_width_height_descent
    font.set_text(s, 0.0, flags=ft2font.LOAD_NO_HINTING)
RuntimeError: In set_text: Could not load glyph (error code 0x14)
```

**Expected outcome**

Docs should build

**Matplotlib version**

  * Operating system: Fedora 30
  * Matplotlib version: 3.2.1
  * Matplotlib backend (`print(matplotlib.get_backend())`):
  * Python version: 3.6.8
  * Other libraries: Sphinx 1.8.2

(Possibly my stuff is outdated; I'll check later.)

**Comment by QuLogic** [comment]
I've been seeing this randomly in CI too now, so I'm not sure it's documentation specific. That's just the easiest/only way I can reproduce.

**Comment by timhoffm** [comment]
@QuLogic Is this still a thing?

**Comment by QuLogic** [comment]
I think this essentially is fixed by #14535 and by virtue of us requiring 3.7 now (since that PR was 3.7 only.)
